CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # App Center SDK for Android Change Log
 
-## Version 1.11.1
+## Version 1.11.1 (Under active development)
+
+### AppCenterAnalytics
+
+* **[Fix]** Don't start a new session from logs in the background.
```

Comment:
Maybe something like "Extend the current session instead of starting a new session when sending events from the background." for better clarity.